fuzzers/forkserver_libafl_cc/src/main.rs

```diff
@@ -185,7 +185,7 @@ pub fn main() {
     if let Some(dynamic_map_size) = executor.coverage_map_size() {
         executor
             .observers_mut()
-            .match_by_ref_mut(observer_ref)
+            .match_by_ref_mut(&observer_ref)
```

Comment:
We could just name this `get`?

Reply:
Yup lol

Reply:
Assuming this will be the normal API to get referencable stuff, going forward